cirq/sim/mux.py

```diff
@@ -183,3 +200,74 @@ def sample_sweep(program: Union[circuits.Circuit, schedules.Schedule],
                               seed=prng)
         trial_results.append(measurements)
     return trial_results
+
+
+def final_density_matrix(
+        program: Union[circuits.Circuit, ops.Gate, ops.OP_TREE, schedules.
+                       Schedule],
+        *,
+        noise: 'cirq.NOISE_MODEL_LIKE' = None,
+        initial_state: Union[int, Sequence[Union[int, float, complex]], np.
+                             ndarray] = 0,
+        param_resolver: study.ParamResolverOrSimilarType = None,
+        qubit_order: ops.QubitOrderOrList = ops.QubitOrder.DEFAULT,
+        dtype: Type[np.number] = np.complex64,
+        seed: Optional[Union[int, np.random.RandomState]] = None
+) -> 'np.ndarray':
+    """Returns the density matrix resulting from simulating the circuit.
+    The terminal measurements would affect the returned result.
```

Comment:
This statement is unclear on its own. Add some context... like

```
"""Returns the density matrix resulting from simulating the circuit.

Note that, unlike `cirq.final_wavefunction`, terminal measurements
are not omitted. Instead, all measurements are treated as sources
of decoherence (i.e. measurements do not collapse, they dephase).
"""
```

Reply:
Done.